src/util.cpp

```diff
@@ -712,7 +712,7 @@ void RenameThread(const char* name)
 #if defined(PR_SET_NAME)
     // Only the first 15 characters are used (16 - NUL terminator)
     ::prctl(PR_SET_NAME, name, 0, 0, 0);
-#elif (defined(__FreeBSD__) || defined(__OpenBSD__))
+#elif (defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__DragonFly))
```

Comment:
[This spec file]](http://ftp.netbsd.org/pub/pkgsrc/current/pkgsrc/lang/gcc46/files/dragonfly-spec.h) seems to imply it's `__DragonFly__`.